frontend/app/professor/simulation-builder/page.tsx

```
@@ -695,7 +765,258 @@ export default function ScenarioBuilder() {
+
+const handleFieldUpdate = (fieldName: string, fieldValue: any) => {
+  console.log('Updating field:', fieldName, fieldValue);
+  
+  switch (fieldName) {
+    case 'title':
+      setName(fieldValue);
+      markAsUnsaved();
+      break;
+    case 'description':
+      const formattedDescription = formatDescription(fieldValue);
+      setDescription(formattedDescription);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)